package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "private": true,
+  "scripts": {
+    "prepare": "lerna run prepare",
```

Comment:
are these scripts covered as well by build and test dependencies CQs ?

Reply:
@benoitf thanks for pointing this out.
lerna isn't covered. I've created a [CQ](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15841).